float32 delta_pleasure # Changes in pleasure
float32 delta_arousal # Changes in arousal
float32 delta_dominance #Changes in dominance